content/zh/blog/_posts/2020-12-02-dockershim-faq.md

```diff
@@ -267,13 +267,13 @@ runtime where possible.
 <!-- 
 Another thing to look out for is anything expecting to run for system maintenance
 or nested inside a container when building images will no longer work. For the
-former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/tasks/debug-application-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
+former, you can use the [`crictl`][cr] tool as a drop-in replacement (see [mapping from docker cli to crictl](https://kubernetes.io/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl)) and for the
 latter you can use newer container build options like [img], [buildah],
 [kaniko], or [buildkit-cli-for-kubectl] that don’t require Docker.
 -->
 另外还有一个需要关注的点，那就是当创建镜像时，系统维护或嵌入容器方面的任务将无法工作。
 对于前者，可以用 [`crictl`](https://github.com/kubernetes-sigs/cri-tools) 工具作为临时替代方案
-(参见 [从 docker 命令映射到 crictl](https://kubernetes.io/zh/docs/tasks/debug-application-cluster/crictl/#mapping-from-docker-cli-to-crictl))；
+(参见 [从 docker 命令映射到 crictl](https://kubernetes.io/docs/tasks/debug/debug-cluster/crictl/#mapping-from-docker-cli-to-crictl))；
```

Comment:
It seems that the "mapping-from-docker-cli-to-crictl" section is moved to https://kubernetes.io/zh/docs/reference/tools/map-crictl-dockercli/ .

So I suggest:

```suggestion
(参见 [从 docker 命令映射到 crictl](/zh/docs/reference/tools/map-crictl-dockercli/))；
```

Reply:
done, thanks for your review @jihoon-seo